linkerd/app/src/env.rs

```diff
@@ -499,7 +499,7 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
             return Err(EnvError::InvalidEnvVar);
         }
 
-        // Ensure that connections thaat directly target the inbound port are
+        // Ensure that connections t directly target the inbound port are
```

Comment:
typo fix has a typo of its own:
```suggestion
        // Ensure that connections that directly target the inbound port are
```